Core/Services/StitchCoreLocalMongoDBService/StitchCoreLocalMongoDBService.podspec

```diff
@@ -39,9 +41,22 @@ Pod::Spec.new do |spec|
   
     spec.source_files = "dist/#{spec.name}/**/*.swift"
 
-    libs = "vendor/MobileSDKs/iphoneos/lib/lib*[^bson-1.0][^mongoc-1.0].dylib"
-    spec.ios.vendored_library = libs
-    spec.tvos.vendored_library = libs
+    def self.vendor_path(platform)
+      Dir.entries("vendor/MobileSDKs/#{platform}/lib/").select {
```

Comment:
[minor] As we discussed, we also want to avoid the copying of duplicate bson/mongoc libraries.